app.js

```diff
@@ -1,12 +1,26 @@
 const express = require("express");
-const { getTopics } = require("./controllers/controllers");
+const { getTopics, getArticles } = require("./controllers/controllers");
 
 const app = express();
 
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles/:article_id", getArticles);
+
 module.exports = app;
 
+app.use((err, request, response, next) => {
+  if (err.code === "22P02") {
+    response.status(400).send({ msg: "Not a valid input" });
+  }
+  next(err);
+});
+
+app.use((err, request, response, next) => {
+    response.status(404).send({ msg: "Input not found" });
+    next(err)
+});
```

Comment:
This error needs a bit more conditional logic, because as-is, all errors that get to this point will be treated as 404s. You've got a promise.reject in your model that would want to be caught here - look into how we handle custom errors, and put in a bit of logic here to do it